nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-service.js

```diff
@@ -1860,6 +1860,16 @@
                     },
                     goToServiceDeferred: function () {
                         return goToServiceFromProperty(serviceTable);
+                    },
+                    getParameterContextId: function (groupId) {
+                        // attempt to identify the parameter context id, conditional based on whether
+                        // the user is configuring the current process group
+                        if (_.isNil(groupId) || groupId === nfCanvasUtils.getGroupId()) {
+                            return nfCanvasUtils.getParameterContextId();
+                        } else {
+                            var parentProcessGroup = nfCanvasUtils.getComponentByType('ProcessGroup').get(groupId);
```

Comment:
`nfCanvasUtils.getComponentByType('ProcessGroup').get(groupId)` only knows about the PG's that are currently rendered on the canvas so if you are in a nested PG that is configured to use the same parameter context of a different nested PG you won't be able to look it up this way...

A potential fix may be that the GoTo for parameter's CS usage may need to actually navigate to the PG before opening the PG's config shell...